src/components/common/ratio/Ratio.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import "twin.macro";
+import "styled-components/macro";
+import Quantity from "../quantity/Quantity";
+import { CQL } from "cqm-models";
+
+interface RatioProps {
+  label: string;
+  numerator: CQL.Quantity;
+  denominator: CQL.Quantity;
+  canEdit: boolean;
+}
+
+const Ratio = ({ label, numerator, denominator, canEdit }: RatioProps) => {
```

Comment:
looks like we need an `onChange` prop here to pass the updated values to the parent. And can we have the input as a single Ratio object (from cql-execution) instead of separate numerator and denominator?

Reply:
Sure.

Reply:
done.